src/shared/components/oncoprint/ResultsViewOncoprint.tsx

```diff
@@ -1045,6 +987,20 @@ export default class ResultsViewOncoprint extends React.Component<IResultsViewOn
         }
     }
 
+    @bind
```

Comment:
autobind?

Reply:
for some reason I thought we were using bind